board/opendingux/package/gmenu2x/gmenu2x-0002-disable-cxx11.patch

```diff
@@ -0,0 +1,11 @@
+--- ./src/Makefile.am.orig	2020-01-31 23:06:50.345918545 +0800
++++ ./src/Makefile.am	2020-01-31 23:05:17.475933881 +0800
+@@ -33,6 +33,7 @@
+ 
+ AM_CXXFLAGS = @CXXFLAGS@ @SDL_CFLAGS@ \
+ 	-fno-exceptions \
+-	-Wall -Wextra -Wundef -Wunused-macros -std=c++11
++	-Wall -Wextra -Wundef -Wunused-macros -std=c++11 \
++	-D_GLIBCXX_USE_CXX11_ABI=0
```

Comment:
@tonyjih @soarqin @martinx72 Any idea why this is necessary?

We build everything from source with the same compiler, so I don't quite understand.
Without this patch, it doesn't work (fails to launch apps).

Reply:
@soarqin has explained it pretty well and I've updated the commit and patch description.

Reply:
> @tonyjih @soarqin @martinx72 Any idea why this is necessary?
> 
> We build everything from source with the same compiler, so I don't quite understand.
> Without this patch, it doesn't work (fails to launch apps).
I updated this because the gkd350h buildroot, which i started, supports GCC 5.4. And this flag is necessary for GCC5.4 to build a proper working gmenu2x.